### PR TITLE
Update to game protocol 12.81

### PR DESCRIPTION
**Pull request by omarcopires**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed 
Support for protocol 12.82 has been added, as well as new effects, outfits and increased market cap.

Cherry-pick from:
https://github.com/Zbizu/forgottenserver/commit/9638dd87b6760bd89a8b17e35dac6448e7765c21
https://github.com/Zbizu/forgottenserver/commit/642aa77eeed63147f0b37c440c0c68ea06486fa7
https://github.com/Zbizu/forgottenserver/commit/3b19c58d449be2e3fa6e28036e5c5489dd74e5a4
https://github.com/Zbizu/forgottenserver/commit/27d5ec3e3f222640a7581acc019241a6197da464
https://github.com/Zbizu/forgottenserver/commit/465a5d022c4daa6cd846640a2adb2fa8cef42bb3
https://github.com/Zbizu/forgottenserver/commit/e0fa6213a087cbed7c78fbfb2cf04845befe9463

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

Co-Authored-By: Zbizu <Zbizu@users.noreply.github.com>
Co-Authored-By: Marcos <66353315+marcosvf132@users.noreply.github.com>
Co-Authored-By: Jonathan Krän <5380287+giuinktse7@users.noreply.github.com>

### CODE REVIEW

File: src/tools.cpp [hunk 1]
```diff
@@ -638,6 +638,9 @@ MagicEffectNames magicEffectNames = {
 	{"orangeteleport",	CONST_ME_ORANGETELEPORT},
 	{"greyteleport",	CONST_ME_GREYTELEPORT},
 	{"lightblueteleport",	CONST_ME_LIGHTBLUETELEPORT},
+	{"fatal", CONST_ME_FATAL},
+	{"dodge", CONST_ME_DODGE},
+	{"hourglass", CONST_ME_HOURGLASS}
```
Comment (DSpeichert) [inline src/tools.cpp:643]: formatting not following pattern above

File: src/itemloader.h [hunk 2]
```diff
@@ -149,6 +150,7 @@ enum itemattrib_t {
 	ITEM_ATTR_WRITEABLE3, //deprecated
 
 	ITEM_ATTR_WAREID,
+	ITEM_ATTR_CLASSLEVEL,
```
Comment (DSpeichert) [inline src/itemloader.h:153]: "CLASSLEVEL" feels redundant, it's "classification", no need for "LEVEL" suffix.
It's class and tier.

**Review by DSpeichert — CHANGES_REQUESTED** [review]
This is really not cherry-picking since you also squashed those commits. It's a PR, so no need to add the unnecessary prefix in the title.

File: src/tools.cpp (hunk 1 above)
Comment (omarcopires) [reply src/tools.cpp]: the formatting appears ok in vscode, but I'm not so sure if it's correct

Comment (MillhioreBT) [reply src/tools.cpp]: I think it refers to this:
https://github.com/otland/forgottenserver/blob/8ca885f7fd38de056072a818e116d791eb85165c/src/tools.cpp#L640

why do you remove the `,`

Comment (omarcopires) [reply src/tools.cpp]: > I think it refers to this:
> 
> https://github.com/otland/forgottenserver/blob/8ca885f7fd38de056072a818e116d791eb85165c/src/tools.cpp#L640
> 
> why do you remove the `,`
> 
> image

corrected, thanks!

**Review by DSpeichert — APPROVED** [review]

File: src/tools.cpp [hunk 3]
```diff
@@ -638,6 +638,9 @@ MagicEffectNames magicEffectNames = {
 	{"orangeteleport",	CONST_ME_ORANGETELEPORT},
 	{"greyteleport",	CONST_ME_GREYTELEPORT},
 	{"lightblueteleport",	CONST_ME_LIGHTBLUETELEPORT},
+	{"fatal", 			CONST_ME_FATAL},
+	{"dodge", 			CONST_ME_DODGE},
+	{"hourglass",		CONST_ME_HOURGLASS},
```
Comment (DSpeichert) [inline src/tools.cpp:643]: This isn't aligned though.

Comment (omarcopires) [reply src/tools.cpp]: I have no idea how to line this up, sorry

Comment (DSpeichert) [reply src/tools.cpp:643]: it may be fine, github displays tabs in a weird way

**Review by DSpeichert — DISMISSED** [review]

**Review by EPuncker — APPROVED** [review]

**Pull request merged**